src/Installation/guides/source.md

```diff
@@ -1,13 +1,13 @@
 # Installation from source
 
 ## Installing Wine
-Before you can install Vinegar, you will first have to install Wine first, you can see the guide for your distribution: [Wine Wiki: Download](https://wiki.winehq.org/Download), you may visit the page according to your preferred Linux distribution of choice. The guide for installing Wine can also be found on your distributions Wiki pages, such as [Arch](https://wiki.archlinux.org/title/wine) and [Gentoo](https://wiki.gentoo.org/wiki/Wine). Additionally, you will need the GnuTLS library in-order for Roblox networking to function properly:
+Before you can install Vinegar, you will first have to install Wine first. You can use the guide for your distributionon the [wine wiki](https://wiki.winehq.org/Download). For instance, you can find guides for [Arch](https://wiki.archlinux.org/title/wine) and [Gentoo](https://wiki.gentoo.org/wiki/Wine). Additionally, you will need the GnuTLS library in-order for Roblox networking to function properly:
 
 - Arch: `lib32-gnutls`
 - Void: `gnutls-32bit` via `void-repo-multilib`
 - Gentoo: `net-libs/gnutls` via `ABI_X86="64 32"`, refer to [Gentoo Wiki: package.use](https://wiki.gentoo.org/wiki//etc/portage/package.use)
 
-It is recommended to make sure you have installed Wine 8.3 and up on your system, as it can give a significant performance increase.
+It is recommended to ensure that you have installed Wine 8.3 or above on your system, as it can give a significant performance increase.
```

Comment:
Add a comment saying that it may be required for Roblox to run too. Roblox needs 8.0 and up and also needs 8.11 (sometimes, maybe) to even run, as it would crash otherwise.

Reply:
added